platform/mbed-trace/source/mbed_trace.c

```diff
@@ -52,7 +52,7 @@
 #define VT100_COLOR_ERROR "\x1b[31m"
 #define VT100_COLOR_WARN  "\x1b[33m"
 #define VT100_COLOR_INFO  "\x1b[39m"
-#define VT100_COLOR_DEBUG "\x1b[90m"
+#define VT100_COLOR_DEBUG "\x1b[94m"
```

Comment:
This may end up into bikeshedding, but did you test with 
```
#define VT100_COLOR_INFO  "\x1b[97m"
#define VT100_COLOR_DEBUG "\x1b[37m"
```
On my eye+terminal combo the end result was readable and it kept text in same color, just different brightness.

No matter what you choose here, someone will not like it. What if one just made these configurable and let one to choose correct colors at build time from their app config?

Reply:
Makes sense in a trace-only world, except that part of the concept is that trace can be interleaved with standard non-trace application output. So that would mean "info trace" would be brighter than non-trace application output.